### ISSUE TITLE
Group and count rows.

### ISSUE THREAD

**Issue by atilaneumann** [JuliaData/DataFrames.jl#556]
Hi,

I tried use "by() function" with more than 200,000 rows and the process is infinite. Did some thing is wrong ?

julia> countResult = by(idxResult, ["column_a", "column_b"], nrow)

Julia closed and I received the message "Segmentation fault (core dumped)".

using DataFrames
csvData = readtable("base.csv", separator = ',', header = true)
aux = join(csvData, csvData, on = :keycolumn, kind = :outer)
result = aux[aux[:column] .!= aux[:column_1], :["column","column_1"]]
idxResult = getindex(result,:["column","column_1"])
csvData = 0
aux = 0
result = 0
gc()
countResult = remotecall_fetch(by(idxResult, ["column", "column_1"], nrow))

Some any idea?

Basically i need count the grouped rows by two columns
